src/Runner.Worker/ActionRunner.cs

```diff
@@ -239,6 +245,8 @@ public async Task RunAsync()
 
             // Run the task.
             await handler.RunAsync(Stage);
+            fileCommandManager.TryProcessFiles(ExecutionContext, ExecutionContext.Global.Container);
```

Comment:
What if the step fail? Do the file-commands still get processed?

For example, esp consider continue-on-error (script exits 1).

Reply:
May need try/finally. If we move into finally, need to make sure doesn't clobber original exception

Reply:
if it fails, set task result failed